### DISCUSSION TITLE
Exact regex implementation isn't specified in the documentation

### DISCUSSION THREAD

**Discussion by mkimi-lightricks** [community/community#9876] [Other Feature Feedback, Questions, & Ideas] [answered]
The documentations about [regex](https://cs.github.com/about/syntax#regex) doesn't specify the exact regex implementation (dialect or feature set if you want). 

Is it [posix](https://en.wikibooks.org/wiki/Regular_Expressions/POSIX_Basic_Regular_Expressions) / [perl](https://perldoc.perl.org/perlre) / basic regex or something else?

**Comment by colinwm** [comment, marked as answer]
The regex engine is based on this one: https://docs.rs/regex/latest/regex/

**Reply by mkimi-lightricks** [reply]
Thanks, consider adding this information to the official documentation.

This data is useful for any user that uses the regex in the code search.
